packages/filesystem/src/browser/file-dialog/file-dialog.ts

```diff
@@ -61,6 +94,8 @@ export class FileDialog extends AbstractDialog<Readonly<FileStatNode> | undefine
 
         this.locationListRenderer = this.createLocationListRenderer();
         navigationPanel.appendChild(this.locationListRenderer.host);
+
+        this.treeFiltersRenderer = this.createFileTreeFiltersRenderer();
```

Comment:
after switching to React, don't forget to dispose `FileDialogTreeFiltersRenderer` similar to `LocationListRenderer`

Reply:
Done